R/tbl_continuous.R

```diff
@@ -94,6 +94,7 @@ tbl_continuous <- function(data,
     digits = digits
   )
 
+  type <- rlang::rep_named(include, list("categorical"))
```

Comment:
@ddsjoberg apparently this also breaks `add_p()` ?!? Can't say I can find the relationship why there

Reply:
nvm it just needed to be modffied and slotted elsewhere